thewalrus/_hafnian.py

```diff
@@ -14,23 +14,20 @@
 """
 Hafnian Python interface
 """
-
+import warnings
 from functools import lru_cache
 from collections import Counter
 from itertools import chain, combinations
 import numba
-
 import numpy as np
 
 
 @numba.jit(nopython=True, cache=True)
 def nb_binom(n, k):  # pragma: no cover
     """Numba version of binomial coefficient function.
-
```

Comment:
Hey @nquesada @dleclerc33 @sduquemesa, I just noticed that a lot of linebreaks were removed from the `_hafnian.py` file docstrings

Unfortunately this will break the documentation, these line breaks are required!

E.g., with the linebreaks:

![image](https://user-images.githubusercontent.com/2959003/153538312-d6e1dc16-a814-4fdf-9eff-179797259081.png)

Without the linebreaks:

![image](https://user-images.githubusercontent.com/2959003/153538370-a69ee13f-1e0c-496c-ad0c-3c1ab7a2e96d.png)

Would someone be able to quickly make a PR to add them back in 🙂

Reply:
By the way, the PR documentation can always be double checked by clicking 'Details' next door to the 'ReadTheDocs' check at the bottom of the PR:

![image](https://user-images.githubusercontent.com/2959003/153538501-31a3c9b3-7539-4c68-8c41-c4ab73caa0a6.png)

This will be updated with every commit to the PR, so will always reflect how the docs look live 🙂